src/iarc7_motion/iarc_tasks/track_roomba_task.py

```diff
@@ -218,16 +224,7 @@ def _check_roomba_in_sight(self):
     def _check_max_roomba_range(self):
         _distance_to_roomba = math.sqrt(self._roomba_point.point.x**2 + 
                             self._roomba_point.point.y**2)
-        
-        return (_distance_to_roomba <= self._MAX_START_TASK_DIST + self._overshoot)
-
-    # checks the radial distance from the drone to the roomba
-    # in order to end the task as successful
-    def _check_max_ending_roomba_range(self):
-        _distance_to_roomba = math.sqrt(self._roomba_point.point.x**2 + 
-                            self._roomba_point.point.y**2)
-        
-        return (_distance_to_roomba <= (self._MAX_END_TASK_DIST + self._overshoot))
+        return (_distance_to_roomba <= self._MAX_START_TASK_DIST)
```

Comment:
Since the meaning of _MAX_START_TASK_DIST has changed we need to change the value in the .param file to be what it was plus the value of the original overshoot parameter.